apps/next-evans/components/core/Stack.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import cx from 'classnames';
+
+interface Props {
+  className?: string;
+  children: JSX.Element[];
+  space: string;
+  el?: 'div' | 'ul';
+  sm?: string;
+  md?: string;
+  lg?: string;
+  xl?: string;
+}
+
+const Stack: React.FC<Props> = ({
```

Comment:
hmmm... forgot about this component. for a little while i was into creating these sort of _layout_ only components, but my hunch is this whole thing can be replaced with `space-y-*` utilities on the parent (which i think weren't available when i made this). can you see if this is easy to eliminate?